pipeline/train/train.sh

```diff
@@ -12,13 +12,18 @@ model_type=$1
 training_type=$2
```

Comment:
Not for now, but this is another script that _might_ benefit from being rewritten in Python (mostly because of all the fun argument handling). https://sh.readthedocs.io/en/latest/ might be useful if you choose to do that. (It's also by no means a requirement as far as I'm concerned.)

Reply:
Yeah, I had a dilemma about that, but I wanted to focus on making this work for now. I think we might give this a try when we'll be integrating the pre-trained OPUS-MT models because it will bring even more complexity.

Reply:
sh lib looks interesting. we should set up a pattern of how we're working with command line tools from Python